faunadb-common/src/main/java/com/faunadb/common/RefAwareHttpClient.java

```diff
@@ -29,13 +28,13 @@ public void close() {
       if (refCount.decrementAndGet() < INITIAL_REF_COUNT && !delegate.isClosed()) {
         delegate.close();
       }
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
Yes, it is.

The try catch was here only to satisfy the compiler since the close method had an `IOException` on it's signature. Even though the `close` method never throw any exception.